### PR TITLE
Removed recurrent logs from getReader

### PR DESCRIPTION
**Pull request by shambhupatil**
Moved debug logs from getReader() to loadScene() and loadGeometry() to reduce the same logs displayed many times.

### CODE REVIEW

File: library/src/loader_impl.cxx [hunk 1]
```diff
@@ -224,8 +224,13 @@ loader& loader_impl::loadGeometry(const std::string& filePath, bool reset)
   }
 
   f3d::reader* reader = f3d::factory::instance()->getReader(filePath);
-  if (!reader)
+  if (reader)
+  {
+    log::debug("The best reader for \"" + filePath + "\" is \"" + reader->getName() + "\"");
```
Comment (mwestphal) [inline library/src/loader_impl.cxx:229]: thinking back on it, maybe it should be something like:

"found a reader for filepath: reader->getName()"

wdyt ?

**Comment by mwestphal** [comment]
not sure what is hapening with codespell, ill take a look

File: library/src/loader_impl.cxx (hunk 1 above)
Comment (shambhupatil) [reply library/src/loader_impl.cxx:229]: Yeah, no need to display reader name then?

**Comment by mwestphal** [comment]
alright I fixed codespell, can you rebase on the last master @shambhupatil ?

**Review by mwestphal — APPROVED** [review]

**Comment by mwestphal** [comment]
all good to me, can I merge @shambhupatil ?

**Pull request merged**